Resources/Locale/en-US/prototypes/catalog/research/technologies.ftl

```diff
@@ -4,6 +4,9 @@ technologies-basic-research-technology-description = Nanotrasen basic research t
 technologies-cleaning-technology = Cleaning technology
 technologies-cleaning-technology-description = Start to a shiny clean station.
 
+technologies-FoodBev-technology = Food and beverage technology
```

Comment:
Looks good from a cursory look however FoodBev doesn't fit the nomenclature of the other localisation strings.

Reply:
Is it the upper case or abbreviation I should change? or both?

Reply:
Well, I lower-cased it. Hopefully, that's what you meant.